implementations/rust/ockam/ockam_command/tests/bats/authority.bats

```diff
@@ -86,7 +86,7 @@ EOF
   run_success bash -c "$OCKAM project import --project-file $OCKAM_HOME/project.json"
 
   # m1 is a member (its on the set of pre-trusted identifiers) so it can get it's own credential
-  run_success "$OCKAM" project enroll --identity m1
+  run_success "$OCKAM" project enroll --identity m1 --output plain
```

Comment:
This change is not necessary, since we are adding any new output format for this command. Removing this change will also make explicit that we are not forcing anyone to use the `plain` format in scripts

Reply:
done